### ISSUE TITLE
Rails project in web/ missing npm package.json

### ISSUE THREAD

**Issue by lazzarello** [melonproject/oyente#263]
Rails server boots but web application throws exception because missing package.json in /oyente/web

```
$ docker run -p 3000:3000 -it oyente:latest /bin/bash
root@1a554292fb82:/oyente/web# ./bin/rails server
Warning: Running `gem pristine --all` to regenerate your installed gemspecs (and deleting then reinstalling your bundle if you use bundle --path) will improve the startup performance of Spring.
=> Booting Puma
=> Rails 5.1.0 application starting in development on http://localhost:3000
=> Run `rails server -h` for more startup options
Puma starting in single mode...
* Version 3.8.2 (ruby 2.3.3-p222), codename: Sassy Salamander
* Min threads: 5, max threads: 5
* Environment: development
* Listening on tcp://0.0.0.0:3000
Use Ctrl-C to stop
Started GET "/" for 172.17.0.1 at 2018-01-30 00:21:44 +0000
Cannot render console from 172.17.0.1! Allowed networks: 127.0.0.1, ::1, 127.0.0.0/127.255.255.255
Processing by HomeController#index as HTML
  Rendering home/index.html.erb within layouts/application
  Rendered home/index.html.erb within layouts/application (2.7ms)
Completed 500 Internal Server Error in 1818ms


  
ActionView::Template::Error (Unable to run node_modules/.bin/browserify. Ensure you have installed it with npm.):
    4:     <title>Oyente - Smart contract security analyzer</title>
    5:     <%= csrf_meta_tags %>
    6: 
    7:     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
    8:     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
    9:   </head>
   10: 
  
app/views/layouts/application.html.erb:7:in `_app_views_layouts_application_html_erb___2841128634323487774_47268273654380'
```

Manually running `npm install` in `/oyente/web`

```
$ docker run -p 3000:3000 -it oyente:latest /bin/bash
root@7456ae630ae8:/oyente/web# npm install
npm WARN saveError ENOENT: no such file or directory, open '/oyente/web/package.json'
npm WARN enoent ENOENT: no such file or directory, open '/oyente/web/package.json'
npm WARN web No description
npm WARN web No repository field.
npm WARN web No README data
npm WARN web No license field.

up to date in 0.092s
```

**Comment by lazzarello** [comment]
this should be fixed in #264

**Comment by lazzarello** [comment]
closed with #264
